### DISCUSSION TITLE
flutter 版本的pag

### DISCUSSION THREAD

**Discussion by AshTerry** [Tencent/libpag#367] [Ideas] [answered]
pag 以后一定行业的新兴标杆，值得每个开发者使用。
目前pag貌似还没有flutter版本，希望能接入比较的流行的跨平台框架。让跨平台开发者也能收益其中。
感谢

**Comment by kevingpqi123** [comment, marked as answer]
我们团队内部自身没有 flutter 的相关业务，不知道什么样的封装对于大家是通用的，大家可以基于 PAG 做一些封装处理，如果是通用的，可以提交 MR 合入 PAG 的仓库中，即便不是通用的，我们也可以帮忙推广
